docs/source/extensions/plugins.rst

```diff
@@ -80,10 +80,7 @@ can then be passed into the Trainer directly or via a (custom) accelerator:
     trainer = Trainer(strategy=CustomDDPPlugin(), plugins=[CustomPrecisionPlugin()])
 
     # fully custom accelerator and plugins
```

Comment:
Custom plugins are dropped from the example?

Reply:
Will the example above sufficient?

Reply:
added